content/en/blog/news/lighthouse-goes-ga.md

```diff
@@ -0,0 +1,41 @@
+
+If you're interested in Jenkins X and Lighthouse support for other SCM providers, please let us know and we'll see what we can do!
+
+### What does "preview support" mean?
+
+The primary difference between Lighthouse's support for GitHub and GitHub Enterprise, and its support for GitLab and BitBucket Server, is testing and usage. The Jenkins X project itself uses Lighthouse heavily, so we're very confident in its behavior and reliability with the GitHub API. Its GitLab and BitBucket Server support is tested with every change to the project, but until we've had reports of Lighthouse users on GitLab or BitBucket Server in real-world usage, we don't want to give the impression that support for those providers is known to be as stable as on GitHub.
+
+So if you're using Jenkins X and Lighthouse with GitLab or BitBucket Server, we'd love to hear how your experience has been, [on our Slack channels or at our weekly office hours](https://jenkins-x.io/community/). If you run into in any problems with Lighthouse on any provider, whether it's in preview or otherwise, please [open an issue](https://github.com/jenkins-x/lighthouse/issues) and we'll look into it as soon as possible. Once we've gotten enough feedback to be confident that major changes will not be needed for GitLab or for BitBucket Server, we'll move those providers out of preview.
+
+### Differences between providers
+
+#### GitHub and GitHub Enterprise
+
+There should be no functional differences between Lighthouse's behavior on github.com vs a GitHub Enterprise instance. The exact same APIs are used in both cases.
+
+#### GitLab
+
+Lighthouse with GitLab may be more likely to hit API rate limits, due to GitLab's GraphQL functionality not matching the particular capabilities of GitHub, but the overall functionality is the same. There is one significant difference with GitLab, however. Some of Lighthouse's ChatOps commands, such as `/approve` and `/assign`, overlap with [GitLab quick actions](https://docs.gitlab.com/ee/user/project/quick_actions.html), and currently, GitLab [does not trigger webhook events for quick actions](https://gitlab.com/gitlab-org/gitlab/-/issues/215934). If Lighthouse does not receive a webhook event, it doesn't have any way of knowing that a ChatOps command has been invoked. Therefore, we have added the ability to invoke [all Lighthouse ChatOps commands](https://jenkins-x.io/docs/build-test-preview/chatops/#chatops-commands) either with the traditional `/approve` etc command, or by adding a `lh-` prefix, like `/lh-approve`. When using Lighthouse with GitLab, you will want to use the `/lh-(whatever)` commands to ensure that your command is actually processed.
```

Comment:
"being the same" instead of "is the same"?

Reply:
Nah, that wouldn't be grammatically correct. =)

Reply:
Maybe. I guess the problem for me is more understanding the causality in this sentence. I guess what you are saying is that with GitLab you need to make more API compared to GitHub to achieve the same functionality and hence you are running more likely into a rate limit. The need to make more API calls, however, is not made explicit. Anyways, ...